test/uk/gov/hmrc/agentaccesscontrol/service/AuthorisationServiceSpec.scala

```diff
@@ -351,39 +351,11 @@ class AuthorisationServiceSpec extends UnitSpec with MockitoSugar {
 
   "isAuthorisedForAfi" when {
 
-    "agent suspension feature flag is OFF" should {
-
-      val agentSuspensionFeatureFlag = false
-
-      "return false when agent suspension feature flag is off and relationships exist" in new Context {
-        whenAgentSuspensionFeatureFlagIsChecked thenReturn agentSuspensionFeatureFlag
-
-        afiRelationshipConnectorIsCheckedForRelatioinships thenReturn Future(
-          true)
-
-        await(
-          authorisationService
-            .isAuthorisedForAfi(agentCode, nino, afiAuthDetails)) shouldBe true
-      }
-
-      "return false when agent suspension feature flag is off and relationships do not exist" in new Context {
-        whenAgentSuspensionFeatureFlagIsChecked thenReturn agentSuspensionFeatureFlag
-
-        afiRelationshipConnectorIsCheckedForRelatioinships thenReturn Future(
-          false)
-
-        await(
-          authorisationService
-            .isAuthorisedForAfi(agentCode, nino, afiAuthDetails)) shouldBe false
-      }
-    }
-
     "agent suspension feature flag is ON" should {
```

Comment:
Please remove this line `"agent suspension feature flag is ON" should {` and update `"isAuthorisedForAfi" when {` to  `"isAuthorisedForAfi" should {` so it looks tidier